torch/quantization/fake_quantize.py

```diff
@@ -155,17 +155,25 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                                                       qscheme=torch.per_tensor_affine,
                                                       reduce_range=True)
 def disable_fake_quant(mod):
-    if type(mod) == FakeQuantize:
+    if type(mod) == FakeQuantize or\
+        isinstance(mod, torch.jit.RecursiveScriptModule)\
+            and mod.original_name == 'FakeQuantize':
```

Comment:
to clarify, why doesn't it work without this code?

Reply:
Since it is scripted, the type will be `class 'torch.jit._script.RecursiveScriptModule` so the old code will return false.